src/components/Forms/CreateTraderForm/CreateTraderForm.scss

```diff
@@ -45,6 +41,42 @@
     }
   }
 
+  .exchangeIconBox {
```

Comment:
I don't understand why adding this new class if there was already iconButton to do the same? What has changed?

Reply:
@cwagner22  the thing changed is that the new icon we added for bitmex is not of the same size, I have manually designed it with black background, and its dimensions are different from other icon, so if we set height and with for icon directly, they differ in dimension, and css doesn't match, so i added a wrapper and styled the wrapper to keep all of them consistent in dimensions.

Reply:
Ok, I think .iconButton can be deleted then?

Reply:
no, it is being used for the other buttons, top buttons, monthly fee and profit sharing.

Reply:
ok, then can we still use the .iconButton class for the exchange icons but add a new .iconExchange with just the few different properties, instead of having most of them repeated, including the selected selector, etc?

Reply:
@cwagner22  Done.